WeatherreportMain.cpp

```diff
@@ -0,0 +1,8 @@
+//#include "Test.h"
+//
+//int main() {
+//

```

Comment:
Why is the code in here commented?

Reply:
I was Testing it local and then forgot to uncomment before commit. Will do it. Thank you.